docs/deployment/upgrade.md

```diff
@@ -8,53 +8,60 @@ hide_title: true
 
-## Cumulus versioning
+## Cumulus Versioning
 
-Cumulus uses a global versioning approach, meaning version numbers are consistent across all packages and tasks, and semantic versioning to track major, minor, and patch version (i.e. 1.0.0). We use Lerna to manage versioning.
+Cumulus uses a global versioning approach, meaning version numbers are consistent across all packages and tasks, and semantic versioning to track major, minor, and patch version (e.g., 1.0.0). We use Lerna to manage versioning.
 
-To view the released module artifacts for each Cumulus core version, see the [Cumulus releases page](https://github.com/nasa/cumulus/releases).
+To view the released module artifacts for each Cumulus core version, see the [Cumulus Releases] page.
 
-**IMPORTANT:** Cumulus releases may introduce breaking changes between minor versions (e.g. 1.13.x -> 1.14.0), so make sure to consult the release notes for migration steps.
+**IMPORTANT:** Cumulus releases may introduce breaking changes between minor versions (e.g., 1.13.x -> 1.14.0), so it is critical that you consult the release notes for migration steps.
 
-## Migrating to a new version
+## Migrating to a New Version
 
-When breaking changes have been introduced, the Cumulus Core team will publish instructions on migrating from one version to another.
+When breaking changes have been introduced, the Cumulus Core team will publish instructions on migrating from one version to another.  Detailed release notes with migration instructions (if any) for each release can be found on the [Cumulus Releases] page.
 
-Detailed release notes with migration instructions (if any) for each release can be found on the [Cumulus releases page](https://github.com/nasa/cumulus/releases).
+**IMPORTANT** 
 
-**IMPORTANT:** When upgrading through many versions, each migration should be done in the order of release (if going from version 1.1.0 from 1.3.0, upgrade from 1.1.0 to 1.2.0 and then to 1.3.0).
+1. **Use consistent Cumulus versions:** All modules must be updated to the same Cumulus version number (see below)
+2. **Follow all intervening steps:** When skipping over versions, you **must perform all intervening migration steps**.  For example, if going from version 1.1.0 to 1.3.0, upgrade from 1.1.0 to 1.2.0 and then to 1.3.0.  This is critical because each release that contains migration steps provide instructions _only_ for migrating from the _immediately_ previous release, but you must follow _all_ migration steps between your currently installed release and _every release_ through the release that you wish to migrate to.
+3. **Migrate lower environments first:** Migrate your "lowest" environment first and test it to ensure correctness before performing migration steps in each successively higher environment.  For example, in the EDCloud progress from a sandbox environment to UAT to SIT to PROD.
```

Comment:
> For example, in the EDCloud progress from a sandbox environment to UAT to SIT to PROD.

This might be confusing to users who don't grok what EDCloud is - can we maybe use a different descriptor?

Reply:
Perhaps it doesn't matter as you already made the point in the previous sentence.

Reply:
Also in the ticket: 

>  guidance on what is a good smoke test? Mention examples of use of EDCloud UAT vs. SIT vs. PROD environments.

I think based on that we need to add content on how to determine things were upgraded properly, or decide explicitly not to add that.

Reply:
I think we can be generic here, like 'before updating your production environment, be sure to update lower testing environments, for example update Sandbox, then SIT, then UAT, the Prod'. Also we should encourage them to do some kind of smoke test. What's a good smoke test? Run through an ingest workflow?

Reply:
I'll remove mention of EDCloud.

Any suggestions on basic smoke tests to insert here?

Reply:
I think run an ingest workflow is good

Reply:
The actuality of a 'smoke test' for a componentized system is .... weird.

The problem with an ingest workflow is a user who has a default deployment....doesn't have one.    But ignoring that, evaluating components: 

- API works
- Workflow run/things appear in api? (yikes in production though) 
- distribution works?
- ....?

I'm not sure I'm advocating for this, just noting it was in the ticket.   We might be fine leaving that up to individual users given the widely divergent configuration options (that get bigger the better we do in moving toward components)